ci/python-package-conda.yml

```diff
@@ -10,14 +10,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.10
-      uses: actions/setup-python@v3
-      with:
-        python-version: '3.10'
     - name: Add conda to system path
       run: |
         # $CONDA is an environment variable pointing to the root of the miniconda directory
         echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install Python
+      run: |
+        conda install python=3.10
```

Comment:
To be clear, it's more common to install Python like that in conda environments like the one provided by installing miniconda in the GitHub action runner default images